marketplacetests/tests/test_marketplace_incorrect_pin.py

```diff
@@ -0,0 +1,44 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from fxapom.fxapom import FxATestAccount
+
+from marketplacetests.marketplace_gaia_test import MarketplaceGaiaTestCase
+from marketplacetests.marketplace.app import Marketplace
+from marketplacetests.payment.app import Payment
+
+
+class TestMarketplaceIncorrectPin(MarketplaceGaiaTestCase):
+
+    def test_incorrect_pin(self):
+
+        APP_NAME = 'Test Zippy With Me'
+        good_pin = '1234'
+        bad_pin = '1111'
```

Comment:
I'd prefer a test_data dictionary (see below example), or at least consistent case in naming.

```python
self.test_data = {
    'app_name': 'Test Zippy With Me',
    'valid_pin': '1234',
    'invalid_pin': '1111'}
```